libs/translator/writer/write_geometry.cpp

```diff
@@ -48,13 +48,19 @@ void UsdArnoldWriteMesh::Write(const AtNode *node, UsdArnoldWriter &writer)
     writer.SetAttribute(mesh.GetOrientationAttr(), UsdGeomTokens->rightHanded);    
     AtArray *vidxs = AiNodeGetArray(node, AtString("vidxs"));
     VtArray<int> vtArrIdxs;
+    bool exportVertices = true;
```

Comment:
Just to be sure, shouldn't exportVertices be initialised to false ? otherwise if vidxs is nulltr then the vertexCount will be exported, may be that is what we want...

Reply:
I think you are right